lzy/allocator/src/main/java/ai/lzy/allocator/alloc/DeleteSessionAction.java

```diff
@@ -78,20 +79,7 @@ private StepResult exec() {
         }
 
         if (vms.isEmpty()) {
-            try {
-                withRetries(log(), () -> {
-                    try (var tx = TransactionHandle.create(allocationContext.storage())) {
-                        completeOperation(null, Any.pack(Empty.getDefaultInstance()), tx);
-                        sessionDao.removeSession(sessionId, tx);
-                        tx.commit();
-                    }
-                });
-            } catch (Exception e) {
-                log().error("{} Cannot complete operation: {}", logPrefix(), e.getMessage(), e);
-                return StepResult.RESTART;
-            }
-            log().info("{} Operation completed", logPrefix());
-            return StepResult.FINISH;
+            return StepResult.CONTINUE;
```

Comment:
it'd be better to return `ALREADY_DONE`

on `CONTINUE` we update operation progress (additional call to DB), on `ALREADY_DONE` we do nothing